File: Programming Language(ver.2)/C++/205.cpp
#include <iostream>
using namespace std;

int main()
{
    int i, num, count[7] = { 0 };
    for (i = 1; i <= 10; i++) {
        cin >> num;
        if (num < 1 || num > 6) {
            count[0]++;
        }
        else {
            count[num]++;
        }
    }
    for (i = 1; i <= 6; i++) {
        cout << "number" << i << ":" << count[i] << endl;
    }
    cout << "error:" << count[0];
    return (0);
}

File: Programming Language(ver.2)/Go/205.go
package main
import "fmt"

func main(){
	var a [10]int
	var record [7]int
	for i := 0; i < 10; i++{
		fmt.Scanln(&a[i])
		if a[i] <= 0 || a[i] > 6{
			record[6] += 1
		}else{
			record[a[i] - 1] += 1
		}
	}
	for i := 0; i < 6; i++{
		fmt.Printf("number%d:%d\n", i+1, record[i])
	}
	fmt.Printf("error:%d", record[6])
}
